app/views/questionnaire.py

```diff
@@ -328,6 +329,13 @@ def get_view_submission(schema, eq_id, form_type):  # pylint: disable=unused-arg
     return redirect(url_for('post_submission.get_thank_you', eq_id=eq_id, form_type=form_type))
 
 
+def _set_started_at_metadata_if_required(form, metadata):
+    questionnaire_store = get_questionnaire_store(current_user.user_id, current_user.user_ik)
+    if not questionnaire_store.answer_store.answers and len(form.data) > 1:
+        logger.info('first answer about to be stored. writing started_at time to metadata')
```

Comment:
any value in logging the start time?

Reply:
whatcha mean sorry?  You mean is it worth logging this line?  or are you saying I should include the started_at time to the log line itself?

Reply:
as in should we also log the time?

Reply:
Yea sure, I'll get that done in a bit.